src/smart-components/ChannelList/utils.js

```diff
@@ -34,6 +34,17 @@ const createEventHandler = ({
         });
       }
     },
+    onUserReceivedInvitation: (channel, inviter, invitees) => {
+      logger.info('ChannelList: onUserReceivedInvitation', { channel, inviter, invitees });
+      const { currentUser } = sdk;
+      const isInvited = invitees.find((user) => user?.userId === currentUser?.userId);
+      if (isInvited) {
+        channelListDispatcher({
+          type: channelActions.USER_INVITED,
+          payload: channel,
+        });
+      }
+    },
     onUserBanned: (channel, user) => {
       const { currentUser } = sdk;
       logger.info('Channel | useHandleChannelEvents: onUserBanned', channel);
```

Comment:
Can you change this log to this?
```suggestion
      logger.info('Channel: onUserBanned', channel);
```

Only this logger has different format here